compiler/rustc_error_messages/locales/en-US/codegen_llvm.ftl

```diff
@@ -11,8 +11,8 @@ codegen_llvm_unknown_ctarget_feature_prefix =
 codegen_llvm_error_creating_import_library =
     Error creating import library for {$lib_name}: {$error}
 
-codegen_llvm_instrument_coverage_requires_llvm_12 =
-    rustc option `-C instrument-coverage` requires LLVM 12 or higher.
+codegen_llvm_instrument_coverage_requires_llvm_13 =
```

Comment:
FWIW this is a pretty poor error message for the end user to see. Most will not be compiling rustc of their own volition and so for them a "the configuration of your rustc build does not support `-C instrument-coverage`" would likely be more actionable (they can complain to whoever built the compiler…)

Reply:
LLVM 13 is our minimum supported version though, so I think this error just doesn't make sense anymore. Should it be replaced with an assertion?

Reply:
The version is also "hardcoded" in the `llvm-wrapper` here:

https://github.com/rust-lang/rust/blob/84462e1174e9c358dbd9889a8bdfbfb7848f4c24/compiler/rustc_llvm/llvm-wrapper/CoverageMappingWrapper.cpp#L105

So its pretty much impossible to compile this with an outdated llvm. I can remove the error message in favor of an assert as well, just to make extra sure the different code paths agree about which version we are targetting.

Reply:
We should probably drop that FFI function as well then? Doesn't seem useful if it always returns the same value (now).